misk/src/main/kotlin/misk/web/MiskWebModule.kt

```diff
@@ -150,7 +151,17 @@ class MiskWebModule(private val config: WebConfig) : KAbstractModule() {
 
   @Provides @Singleton
   fun provideJettyThreadPool(): QueuedThreadPool {
-    return config.jetty_max_thread_pool_size?.let { QueuedThreadPool(it) } ?: QueuedThreadPool()
+    val maxThreads = config.jetty_max_thread_pool_size
+    val minThreads = Math.min(8, maxThreads)
+    val idleTimeout = 60_000
+
+    val threadPool = QueuedThreadPool(
+        maxThreads,
+        minThreads,
+        idleTimeout,
+        ArrayBlockingQueue(config.jetty_max_thread_pool_queue_size))
+    threadPool.name = "jetty-thread"
```

Comment:
lol this can't be provided int he constructor?

Reply:
No, but you can pass in a ThreadGroup with a name but that seems different than the actual name value that gets set in the `QueuedThreadPool`